src/buza/views.py

```diff
@@ -48,6 +47,32 @@ def register(request: HttpRequest) -> HttpResponse:
     )
 
 
+class UserCreate(generic.CreateView):
+    model = models.User
+    fields = [
+        'username', 'email', 'phone',
+    ]
+
+    def dispatch(
+            self,
+            request: HttpRequest,
+            *args: Any,
+            **kwargs: Any,
+    ) -> HttpResponse:
+        """
+        Look up the question, and set `self.question`.
+        """
+        assert not request.user.is_authenticated, request.user
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_success_url(self) -> str:
+        """
+        Redirect to the question.
+        """
+        success_url: str = reverse('question-list')
+        return success_url
```

Comment:
We can rather finish this conversion in a follow-up branch, I think, and keep using the existing `register` view, for now.